content/events-api/subscription.md

```diff
@@ -76,3 +77,40 @@ This request is JSON-based (`Content-Type: application/json`) and contains wrapp
 | `pim_source` | Identify which Akeneo PIM sent the event. | `demo.akeneo.com` | From `AKENEO_PIM_URL` environment variable |
 | `data` | Contains the resource data. Learn more about the data wrapper, including its JSON schema, on our [Events API Reference](/events-reference/events-reference-serenity/products.html). | | |
 
+
+## Debugging events
+
+![Download logs button](../img/events-api/connection-download-logs-button.png)
+
+### Log levels
+As you can read on our Help center page [Subscribe and manage **events**](https://help.akeneo.com/pim/serenity/articles/manage-event-subscription.html#debug), the log file gives you access to 4 log levels: 
+- **Error** logs show you when something went wrong with the connected application. 
```

Comment:
**If** we want to expand on the levels, maybe we should take about the meaning of each level.

(I've tried to look for good ones, and did not succeed)

I think that the logs types you describe in the next part are enough because they show what they can expect for each level.